pkg/repository/udmrepo/repo.go

```diff
@@ -68,6 +68,7 @@ type ObjectWriteOptions struct {
 	Prefix      ID     // A prefix of the name used to save the object
 	AccessMode  int    // OBJECT_DATA_ACCESS_*
 	BackupMode  int    // OBJECT_DATA_BACKUP_*
+	AsyncWrites int    // Num of async writes for the object, 0 means no async write
```

Comment:
Looks like there is no place to assign value to the `AsyncWrites` parameter.

Reply:
Fixed in `(sr *shimRepository) NewObjectWriter`, it is an optional value, so right now, we don't need to set a value, just convey the value from shim to unified repo.